processing file asmFiles/associativity.asm (1 of 21)
processing file asmFiles/branch.asm (2 of 21)
processing file asmFiles/count_days.asm (3 of 21)
--- memsim.hex	2021-10-27 20:09:25.296940860 -0400
+++ memcpu.hex	2021-10-27 20:09:25.176936510 -0400
@@ -50,6 +50,6 @@
 :0400310027BDFFFCEC
 :04003200AFBC00005F
 :0400330003E00008DE
-:043FFD0000000015AB
-:043FFE0000001EE6BB
+:043FFD00000002625C
+:043FFE000000001EA1
 :00000001FF
processing file asmFiles/fib.asm (4 of 21)
--- memsim.hex	2021-10-27 20:09:27.233011040 -0400
+++ memcpu.hex	2021-10-27 20:09:27.145007850 -0400
@@ -12,27 +12,16 @@
 :04000B0015E0FFF904
 :04000C00FFFFFFFFF4
 :0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
+:0400230000000001D8
+:0400250000000002D5
+:0400270000000004D1
+:0400280000000004D0
+:04002A0000000008CA
+:04002C0000000010C0
+:04002E0000000020AE
+:04003000000000408C
+:04003200000000804A
+:0400340000000100C7
+:0400360000000200C4
 :0403C0000000001623
 :00000001FF
processing file asmFiles/halt.asm (5 of 21)
processing file asmFiles/hit_diff.asm (6 of 21)
processing file asmFiles/hit_same.asm (7 of 21)
processing file asmFiles/jump.asm (8 of 21)
processing file asmFiles/mergesort.asm (9 of 21)
--- memsim.hex	2021-10-27 20:09:39.545457348 -0400
+++ memcpu.hex	2021-10-27 20:09:39.430453179 -0400
@@ -82,132 +82,103 @@
 :0400510003E00008C0
 :0400C00000000040FC
 :0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
+:0400C2000000000733
+:0400C3000000000B2E
+:0400C4000000005ADE
+:0400C500000000191E
+:0400C60000000050E6
+:0400C70000000050E5
+:0400C800000000290B
+:0400C900000000151E
+:0400CA000000000C26
+:0400CB000000003EF3
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000053DB
+:0400CF0000000051DC
+:0400D00000000051DB
+:0400D10000000051DA
+:0400D2000000001614
+:0400D3000000001D0C
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E1000000000813
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F40000000017F1
+:0400F5000000005CAB
+:0400F6000000005BAB
+:0400F70000000028DD
+:0400F8000000001AEA
+:0400F90000000008FB
+:0400FA0000000024DE
+:0400FB0000000026DB
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
 :0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
+:0401410000000007B3
+:0401420000000008B1
+:040143000000000BAD
+:040145000000005A5C
+:040147000000000FA5
+:04014800000000199A
+:040149000000004171
+:04014A000000002091
+:04014B0000000009A7
+:04014C000000002F80
+:04014D00000000505E
+:04014E00000000119C
+:04014F0000000007A5
+:040150000000002388
+:040151000000004466
+:040152000000002089
+:040153000000000A9E
+:0401540000000007A0
+:04015500000000178F
+:04015600000000178E
+:040157000000005054
+:04015800000000297A
+:04015A00000000158C
+:04015B000000000C94
+:04015D000000000896
 :04015E000000002479
 :04015F000000002676
 :040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:04016100000000108A
+:040162000000003267
+:040163000000000791
+:040164000000003E59
+:04016600000000474E
+:040167000000005341
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (10 of 21)
--- memsim.hex	2021-10-27 20:09:41.544529811 -0400
+++ memcpu.hex	2021-10-27 20:09:41.410524954 -0400
@@ -31,5 +31,5 @@
 :04001E00AFBC000073
 :04001F00FFFFFFFFE1
 :043FFD0000000012AE
-:043FFE00000001328C
+:043FFE0000000011AE
 :00000001FF
processing file asmFiles/mult_procedure.asm (11 of 21)
processing file asmFiles/test.branch.asm (1 of 7)
processing file asmFiles/test.branchjump1.asm (2 of 7)
processing file asmFiles/test.jump.asm (3 of 7)
--- memsim.hex	2021-10-27 20:11:24.081246611 -0400
+++ memcpu.hex	2021-10-27 20:11:23.950241862 -0400
@@ -1,4 +1,3 @@
-:0400000008000006EE
 :0400010000FF0000FC
 :04000200BBFF000040
 :04000300BEEF00004C
processing file asmFiles/test.ldst1.asm (4 of 7)
processing file asmFiles/search.asm (1 of 1)
processing file asmFiles/test.ldst2.asm (1 of 1)
processing file asmFiles/test.loadstore.asm (1 of 1)
processing file asmFiles/test.rtype.asm (1 of 1)
--- memsim.hex	2021-10-27 20:13:30.378824500 -0400
+++ memcpu.hex	2021-10-27 20:13:30.272820658 -0400
@@ -29,13 +29,9 @@
 :04001C00FFFFFFFFE4
 :040020000000F7F9EC
 :040021000000126168
-:0400220000000009D1
-:0400230000010A5A74
 :0400240000007F3920
 :04002500FFFFDA708F
 :04002600000037F8A7
-:04002700000021565E
-:04002800000D269011
 :04002900000006933A
 :04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/unit_test_load.asm (1 of 2)
processing file asmFiles/unit_test_raw.asm (2 of 2)
